terraform/aws/env/us-east/variables.tf

```diff
@@ -11,19 +11,25 @@ variable "region" {
   default     = "us-east-1"
 }
 
+variable "USE_HCP_PACKER"{
+  default     = true
+}
+
 variable "ami" {
+  default ="12345"
```

Comment:
We probably shouldn't ever have a variable default that can never be correct.

Reply:
To address this, and the larger question of context from the master comment:  

My idea was to add a toggle to allow one to switch between HCP/non-HCP.  I put HCP as the default b/c that was the new functionality I was testing. I can switch it.  

 On to the variable issue.    In the case of HCP packer, that variable will go unused but needs some kind of value in order for Terraform to accept it.  In the non-HCP case, the user needs to set the value to whatever their AMI turned out to be.  I need some kind of placeholder and I could use a 'real' AMI but unless that AMI is correctly built it's going to be an "incorrect default value" which is the condition of the current code from the .json template.  My thinking was to put in a fake value as the default so that the apply will fail immediately if the user did not follow the instructions in the README.  Maybe the comment I really need to add is one saying "this is fake. replace it with your own if you aren't using HCP"?

Reply:
> In the case of HCP packer, that variable will go unused but needs some kind of value in order for Terraform to accept it.

Shouldn't both the HCP and local Packer workflows be able to tag the AMI such that we could query for it as a data source, rather than a variable? Then there's no fiddly copy-and-pasting around the AMI ID or doing things conditional on the chosen workflow.

In any case, having a value that's a "real value" but isn't in the right format is misleading. Does using `""` cause it to blow up?